_sass/mei.scss

```diff
@@ -393,11 +393,21 @@ ul.nav li.nav-item:first-child {
         content: "Goal: "
     }
 
+    .tutorialGoalES:before {
```

Comment:
```suggestion
    .tutorialGoal-ES:before {
```

Reply:
Suggestion to use hyphen as in file names ("tutorials-ES.html"). Would need to be adjusted in all classes in html files.

Reply:
Also, the `tutorialGoal-ES` selector needs to be added above with the `.tutorialGoal` rule, like:

```
 .tutorialGoal, .tutorialGoal-ES {
        margin-top: .5rem;
}

```